compilers/concrete-compiler/compiler/lib/Runtime/StreamEmulator.cpp

```diff
@@ -3,23 +3,26 @@
 // https://github.com/zama-ai/concrete-compiler-internal/blob/main/LICENSE.txt
 // for license information.
 
+#include <concretelang/Runtime/stream_emulator_api.h>
+#include <concretelang/Runtime/wrappers.h>
 #include <cstdarg>
 #include <iostream>
 #include <memory>
 #include <mutex>
 #include <numeric>
 #include <queue>
+#include <sched.h>
 #include <thread>
 #include <utility>
 #include <vector>
 
-#include <sched.h>
-
-#include <concretelang/ClientLib/Types.h>
-#include <concretelang/Runtime/stream_emulator_api.h>
-#include <concretelang/Runtime/wrappers.h>
-
-using concretelang::clientlib::MemRefDescriptor;
+template <size_t N> struct MemRefDescriptor {
```

Comment:
Same definition in `GPUDFG.cpp`. Maybe move to a header file?

Reply:
Done !